connector/mqtt/locust/README.md

```diff
@@ -0,0 +1,376 @@
+# **Dojot Load Test**
+The Dojot Load Test tool is an implementation using the open-source load testing tool Locust to generate traffic
+from/to Dojot.
+
+
+# **Configuration**
```

Comment:
I'd like to read "how it works" before "configuration". This is just a personal opinion.